js/src/dropdown.js

```diff
@@ -416,35 +421,39 @@ const Dropdown = (() => {
    * ------------------------------------------------------------------------
    */
 
-  $(document)
-    .on(Event.KEYDOWN_DATA_API, Selector.DATA_TOGGLE,  Dropdown._dataApiKeydownHandler)
-    .on(Event.KEYDOWN_DATA_API, Selector.MENU, Dropdown._dataApiKeydownHandler)
-    .on(`${Event.CLICK_DATA_API} ${Event.KEYUP_DATA_API}`, Dropdown._clearMenus)
-    .on(Event.CLICK_DATA_API, Selector.DATA_TOGGLE, function (event) {
-      event.preventDefault()
-      event.stopPropagation()
-      Dropdown._jQueryInterface.call($(this), 'toggle')
-    })
-    .on(Event.CLICK_DATA_API, Selector.FORM_CHILD, (e) => {
-      e.stopPropagation()
-    })
+  EventHandler.on(document, Event.KEYDOWN_DATA_API, Selector.DATA_TOGGLE, Dropdown._dataApiKeydownHandler)
+  EventHandler.on(document, Event.KEYDOWN_DATA_API, Selector.MENU, Dropdown._dataApiKeydownHandler)
```

Comment:
@alekitto they are an issue here with our current EventHandler because if we have the same delegation handler for a different delegation selector no event listener is added due to that line : https://github.com/twbs/bootstrap/blob/v4-without-jquery/js/src/dom/eventHandler.js#L225

Reply:
Fixed by my last commit 👍

Reply:
Yes, you're right, I managed to reproduce the error in a test case. PR is coming to fix this.